Purchases/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -152,6 +152,10 @@ internal protocol StoreProductType {
     var introductoryDiscount: StoreProductDiscount? { get }
 
     /// An array of subscription offers available for the auto-renewable subscription.
+    /// - Note: the currenet user may or may not be eligible for some of these.
+    /// - Seealso: ``Purchases/checkPromotionalDiscountEligibility(forProductDiscount:product:)``
+    /// - Seealso: ``Purchases/checkPromotionalDiscountEligibility(forProductDiscount:product:completion:)`
```

Comment:
missing one backtick at the end of this line

Reply:
Oops thanks